src/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div id="root">
+        Not rendered
+        <div id="content"></div>
+    </div>
+    <script type="module" src = "App.js"></script>
```

Comment:
use prettier for uniform formatting. Also can use defer to move this script up above in the head.